examples/gateway/pix_payment.php

```diff
@@ -7,22 +7,21 @@
 include __DIR__ . '/../../vendor/autoload.php';
 
 try {
-    $pixPayment = new \VendoSdk\Gateway\PixPayment();
-    $pixPayment->setApiSecret('your_secret_api_secret');
-    $pixPayment->setMerchantId(1);//Your Vendo Merchant ID
-    $pixPayment->setSiteId(1);//Your Vendo Site ID
+    $payment = new \VendoSdk\Gateway\Payment();
+    $payment->setApiSecret('your_secret_api_secret');
+    $payment->setMerchantId(1);//Your Vendo Merchant ID
+    $payment->setSiteId(85133);//Your Vendo Site ID
 
-    $pixPayment->setAmount(10.50);
-    $pixPayment->setCurrency(\VendoSdk\Vendo::CURRENCY_USD);
-    $pixPayment->setIsTest(true);
+    $payment->setAmount(10.50);
+    $payment->setCurrency(\VendoSdk\Vendo::CURRENCY_USD);
+    $payment->setIsTest(true);
 
     //You must set the flag below to TRUE if you're processing a recurring billing transaction
```

Comment:
Recurring billing is not supported by PIX, so let's not mention it here?

Reply:
ah ok ;)

Reply:
removed. Also from response parser - end of example file